sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -151,13 +143,15 @@ class RapidsDriverPlugin extends DriverPlugin with Logging {
     }
   }
 
-  override def init(sc: SparkContext, pluginContext: PluginContext): util.Map[String, String] = {
+  override def init(
+    sc: SparkContext, pluginContext: PluginContext): java.util.Map[String, String] = {
     val sparkConf = pluginContext.conf
     RapidsPluginUtils.fixupConfigs(sparkConf)
     val conf = new RapidsConf(sparkConf)
-    if (conf.shimsProviderOverride.isDefined) {
+    if (conf.shimsProviderOverride.isDefined) { // TODO test it, probably not working yet
```

Comment:
Have you tested this yet?

Reply:
not yet

Reply:
actually hard to test before I can pull @tgravescs's change